Week-1/Assignment 1.2/style.css

```diff
@@ -0,0 +1,6786 @@
+.first-section{
+    display: flex;
+    align-items: flex-start; 
+    
+}
+#img1, #img2, #img3 {
+    width: 30px;
+    margin-right: 2px;
+}
+
+h6#head2 {
+  position: relative;
+  top: -14px;
+}
+
+.playbox{
+    display: flex;
+    align-items: center;
+}
+header.header_section {
+    padding: 20px;
+    background-color: rgb(30, 29, 29);
+}
+
+.headline {
+  color: white;
+}
+
+i.las.la-play {
+  position: relative;
+  top: 5px;
+}
+
+i.las.la-plus {
+  position: relative;
+  top: 4px;
+}
+
+i.las.la-ellipsis-h {
+  position: relative;
+  top: 5px;
+}
+
+.imageplustext {
+  display: flex;
+  align-items: center;
+  position: relative;
+  top: 62px;
+}
+
+.footericon {
+  display: flex;
+  align-items: center;
+  position: relative;
+  right: -1220px;
+}
+
+i.las.la-pause {
+  position: relative;
+  right: -54px;
+}
+
+i.las.la-step-forward {
+  position: relative;
+  right: -108px;
+}
+
+i.las.la-undo-alt {
+  position: relative;
+  right: -162px;
+}
+
+i.las.la-random {
+  position: relative;
+  right: -216px;
+}
+
+.icon {
+    font-size: 15px;
+    line-height: 10px;
+    color: white;
+    margin-right: 20px;
+}
+
+.text_img {
+    margin-left: 43px;
+}
+table {
+    font-family: Lato;
+    border-collapse: collapse;
+    width: 96%;
+    margin-left: 10px;
+  }
+
+  td, th {
+    border: 1px solid #dddddd;
+    text-align: left;
+    padding: 8px;
+  }
+  tr:nth-child(even) {
+    background-color: #dddddd;
+  }
+  .last_section{
+    padding: 20px;
+    background-color: rgb(111, 111, 217);
+    display: flex;
+    
+  }
+  #footerimg {
+    width: 100px;
+    
+  }
+  .ftext {
+    margin-left: 20px;
+}
+
+body {font-family: "Lato", sans-serif;}
+
+.sidebar {
+  height: 70%;
+  width: 20px;
+  position: fixed;
+  z-index: 1;
+  top: 0;
+  left: 0;
+  background-color: white;
+  overflow-x: hidden;
+  padding-top: 16px;
+}
+
+.sidebar {
+  padding: 6px 8px 6px 16px;
+  text-decoration: none;
+  font-size: 20px;
+  color: #766464;
+  display: block;
+}
+
+.sidebar :hover {
+  color: #1818af;
+}
+
+header, table {
+  margin-left: 50px; /* Same as the width of the sidenav */
+  
+}
+
+i{
+    margin-bottom: 12px;
+}
+
+.footer {
+    position: fixed;
+    left: 0;
+    bottom: 0;
+    width: 100%;
+    background-color: rgb(56, 18, 172);
+    color: white;
+    text-align: center;
+}
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+.la,
```

Comment:
If these are from a 3rd party library, let's have it in another separate file and import it instead of having it all in the same file.

Reply:
Since I am already exporting line-awesome css style sheet. This section is no more required. Have removed it.